### DISCUSSION TITLE
Log in ssh terminal

### DISCUSSION THREAD

**Discussion by dalogue1** [Nachtzuster/BirdNET-Pi#151] [General]
Does anyone know what the command would be to have the BirdNET-Pi log in an ssh terminal window?

I typically have an ssh terminal logged into the BirdNET-Pi system and often would like to see the running log for the system.  I then need to go under the View Log menu item.  Is there a command that would allow the log files to scroll along in a terminal window using ssh.

**Comment by baloor666** [comment]
journalctl -f

**Reply by dalogue1** [reply]
Thank you.  This is just what I was looking for.
